.chloggen/kafka-exporter-aws-iam-oauth-bearer.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: kafkaexporter, internal/kafka
```

Comment:
isn't receiver missing here?

Reply:
I guess we can remove the internal/kafka?

Reply:
Thanks. I updated the changelog.